build.sh

```diff
@@ -29,12 +29,22 @@ fi
 
 if [ $(DOCKER_CLI_EXPERIMENTAL=enabled docker manifest inspect $build_image_name 2> /dev/null ; echo $?) -eq 0 ]; then
 	echo "----> Skipping build as image already exists"
-else
+	exit 0
+fi
+
+if [ ! -z $MAESTRO_BUILD_FOLDER ]; then
+	cd $MAESTRO_BUILD_FOLDER
```

Comment:
Maybe instead of changing directory you could leverage the `docker` and `pack` path parameters